### ISSUE TITLE
No root access

### ISSUE THREAD

**Issue by bytesex** [rogerb831/Yi-Home-Allwinner-Root#1]
Hi,
Currently I have the Yi 1080p Home camera with a firmware of 8.2.0.0A_201912121707

Did all the steps you mentioned and ended up with constantly flashing blue indicator.
Cannot telnet to cam.
Maybe Im doing something wrong?

Is it okay that factory_test.sh is empty?
After I removed the sd card, my cam was (reverted?) to and old state. The app is working

**Comment by rogerb831** [comment]
Should just work. Do you have a serial adapter to see the messages?

To confirm, you started with a working camera and an SD card that it can read, then put the hack on the SD, then reboot, and constant blinking blue is the result? Did the Factory folder get renamed on the SD card?

**Comment by bytesex** [comment]
I tried to rerun my actions - I need to state that I followed the instructions on https://github.com/roleoroleo/yi-hack-Allwinner a day ago. I did the following:
Root your Camera
Format an SD Card as FAT32
Copy the contents of sdhack/ to the root of the SD Card
Insert the SD Card and reboot the camera
The camera is no longer checking signatures for installation images.

Thats all. Neither that, nor your version of 'Factory' folder gets renamed.
You mentioned a serial adapter - what kind of adapted should I have? 
And how do I reach the camera via telnet? Maybe it was rooted, but I have no sense of it =)
